tests/test_coordinator.py

```diff
@@ -513,119 +496,121 @@ def test_post_aggregation_fails(
-    for node in nodes:
-        coordinator.postTranscript(0, transcript, sender=node)
-
-    aggregated = transcript
-    decryption_request_static_keys = [os.urandom(42) for _ in nodes]
-    dkg_public_key = (os.urandom(32), os.urandom(16))
-    for i, node in enumerate(nodes):
-        coordinator.postAggregation(
-            0, aggregated, dkg_public_key, decryption_request_static_keys[i], sender=node
-        )
-
-    # Actually authorize
-    tx = global_allow_list.authorize(0, [deployer.address], sender=initiator)
-
-    # Authorized
-    assert global_allow_list.isAuthorized(0, bytes(signature), bytes(data))
-    assert coordinator.isEncryptionAuthorized(0, bytes(signature), bytes(data))
-    events = global_allow_list.AddressAuthorizationSet.from_receipt(tx)
-    assert events == [
-        global_allow_list.AddressAuthorizationSet(
-            ritualId=0, _address=deployer.address, isAuthorized=True
-        )
-    ]
-
-    # Deauthorize
-    tx = global_allow_list.deauthorize(0, [deployer.address], sender=initiator)
-
-    assert not global_allow_list.isAuthorized(0, bytes(signature), bytes(data))
-    assert not coordinator.isEncryptionAuthorized(0, bytes(signature), bytes(data))
-    events = global_allow_list.AddressAuthorizationSet.from_receipt(tx)
-    assert events == [
-        global_allow_list.AddressAuthorizationSet(
-            ritualId=0, _address=deployer.address, isAuthorized=False
-        )
-    ]
-
-    # Reauthorize in batch
-    addresses_to_authorize = [deployer.address, initiator.address]
-    tx = global_allow_list.authorize(0, addresses_to_authorize, sender=initiator)
-    signed_digest = w3.eth.account.sign_message(signable_message, private_key=initiator.private_key)
-    initiator_signature = signed_digest.signature
-    assert global_allow_list.isAuthorized(0, bytes(initiator_signature), bytes(data))
-    assert coordinator.isEncryptionAuthorized(0, bytes(initiator_signature), bytes(data))
-
-    assert global_allow_list.isAuthorized(0, bytes(signature), bytes(data))
-    assert coordinator.isEncryptionAuthorized(0, bytes(signature), bytes(data))
-
-    events = global_allow_list.AddressAuthorizationSet.from_receipt(tx)
-    assert events == [
-        global_allow_list.AddressAuthorizationSet(
-            ritualId=0, _address=deployer.address, isAuthorized=True
-        ),
-        # TODO was this originally supposed to True (not sure why it passed before)
-        global_allow_list.AddressAuthorizationSet(
-            ritualId=0, _address=initiator.address, isAuthorized=True
-        ),
-    ]
+    assert refund == fee - fee_model.feeDeduction(pending_fee, DURATION)
+    assert fee_model_balance_after_refund + refund == fee
+    assert fee_model.totalPendingFees() == 0
+    assert fee_model.pendingFees(ritualID) == 0
+    fee_model.withdrawTokens(fee_model_balance_after_refund, sender=deployer)
+
+
+# def test_authorize_using_global_allow_list(
```

Comment:
We can delete all theses tests since they have been moved to test_global_allow_list.py, right?

Reply:
yep, forgot to do that, thanks

Reply:
✔️